sharding/indexHashedNodesCoordinator.go

```diff
@@ -553,11 +557,13 @@ func (ihgs *indexHashedNodesCoordinator) EpochStartPrepare(metaHdr data.HeaderHa
 	err = ihgs.setNodesPerShards(resUpdateNodes.Eligible, resUpdateNodes.Waiting, leavingNodesMap, newEpoch)
 	if err != nil {
 		log.Error("set nodes per shard failed", "error", err.Error())
+		return
 	}
 
 	err = ihgs.saveState(randomness)
 	if err != nil {
 		log.Error("saving nodes coordinator config failed", "error", err.Error())
+		return
```

Comment:
even if there was a store error, the configuration was already changed so I would not return, but leave it to display the configuration for debugging and clear the consensus group cache.

Reply:
yes. removed return